packages/snap-controller/src/Search/README.md

```diff
@@ -13,6 +13,7 @@ The `SearchController` is used when making queries to the API `search` endpoint.
 | settings.facets.pinFiltered | selected facet options move to the top of the options array | true |   |
 | settings.facets.storeRange | range facets keep their initial boundary ranges from first load | true |   |
 | settings.facets.trim | facets that do not change results will be removed | true |   |
+| settings.facets.disableAutoCollapsedHandling | disable "auto open" functionality for facets that are filtered (active), collapsed, and have no stored data | false |   |
```

Comment:
I'm sure you saw this coming - but can we rename this?

disableAutoCollapsedHandling -> autoOpenActive

The default value should be true - but can be set to false via this config.